css/css-animations/KeyframeEffect-getKeyframes.tentative.html

```diff
@@ -113,6 +125,28 @@
   to   { margin-top: 20px; margin-right: 20px; margin-bottom: 20px; }
 }
 
+@keyframes anim-merge-offset-and-composite {
```

Comment:
I think this test makes sense although I couldn't find where in the specs it's defined that animation-composition participates in the keyframe merging behavior like animation-timing-function does.

Reply:
I've filed https://github.com/w3c/csswg-drafts/issues/6935.